### PR TITLE
Preview output noise frequency of seeds

### PR DESCRIPTION
**Pull request by 4D42**
- Save the noise frequency of seeds. In the hdf5 preview file.
- add more definitions
- modification of variable names

### CODE REVIEW

File: MiniAn/minian_main.py [hunk 1]
```diff
@@ -517,7 +535,7 @@ def save_minian_to_doric(
     res = Y - AC # residual images
 
     duration = Y.shape[0]
-    time_ = np.arange(0, duration/fr, 1/fr, dtype="float64")
+    time_ = np.arange(0, duration/freq, 1/freq, dtype="float64")
```
Comment (4D42) [inline MiniAn/minian_main.py:537]: @doriclenses  Do I also load time from file here too?
I could add `time` as parameter and load it just before calling save_minian_to_doric.
Or I could load it on the fly there

Comment (doriclenses) [reply MiniAn/minian_main.py:537]: yepp

File: MiniAn/minian_main.py [hunk 2]
```diff
@@ -555,28 +571,28 @@ def save_minian_to_doric(
 
         print(mn_defs.Messages.SAVE_ROI_SIG, flush=True)
         pathROIs = vpath+mn_defs.DoricFile.Group.ROISIGNALS+operationCount+'/'
-        utils.save_roi_signals(C.values, A.values, time_, f, pathROIs+vdataset, attrs_add={"RangeMin": 0, "RangeMax": 0, "Unit": "AU"})
```
Comment (doriclenses) [inline MiniAn/minian_main.py:558]: I think `time` is reserved for something else in python, so I would be careful to remove that `_`

Comment (4D42) [reply MiniAn/minian_main.py:558]: Ok I will check that

File: MiniAn/minian_main.py [hunk 3]
```diff
@@ -133,21 +135,38 @@ def preview(minian_parameters):
...
 
-            if mn_defs.Preview.Dataset.MAX_PROJECTION in hdf5_file:
-                del hdf5_file[mn_defs.Preview.Dataset.MAX_PROJECTION]
-
-            hdf5_file.create_dataset(mn_defs.Preview.Dataset.MAX_PROJECTION, data = max_proj.values, dtype = "float64", chunks = True)
-
-            seeds_dataset = hdf5_file.create_dataset(mn_defs.Preview.Dataset.SEEDS, data = seeds[["width", "height"]], dtype="int", chunks = True)
+            seeds_dataset = initialization_group.create_dataset(mn_defs.Preview.Dataset.SEEDS, data = seeds[["width", "height"]], dtype="int", chunks = True)
             seeds_dataset.attrs[mn_defs.Preview.Attribute.MERGED]  = seeds.index[seeds["mask_mrg"] == True].tolist()
             seeds_dataset.attrs[mn_defs.Preview.Attribute.REFINED] = seeds.index[(seeds["mask_ks"] == True) & (seeds["mask_pnr"] == True)].tolist()
```
Comment (doriclenses) [inline MiniAn/minian_main.py:160]: Please also save "SeedsCount"

**Review by AntoineLet — APPROVED** [review]

**Pull request merged**